app/view_models/nsm/v1/letter_and_call.rb

```diff
@@ -114,7 +114,7 @@ def provider_fields
 
         if vat_registered?
           rows['.vat'] = NumberTo.percentage(vat_rate)
-          rows['.total_claimed_inc_vate'] = NumberTo.pounds(provider_requested_amount_inc_vat)
+          rows['.total_claimed_inc_vat'] = NumberTo.pounds(provider_requested_amount_inc_vat)
```

Comment:
We now have a ticket I'm working on to actually remove this logic entirely - as we only calculate VAT once all work items etc have been summed up, showing 'individual VAT' is misleading. So I think this PR is superseded by #737

Reply:
Joel's off today so either add "Closes #735" to your PR or we can close this 😄